docs/concepts/collection.md

```diff
@@ -10,25 +10,44 @@ Collections are groups of related content that can be viewed or managed as a uni
+- a "Children" tab on resources, which provides a management interface to access, re-order, add, or delete the members of a resource based on the Member Of field.
+- a "Model" field (`field_model`) which can take various values including "Collection".
 
 ## Islandora Defaults features
 
 [Islandora Defaults](../reference/islandora_defaults_reference.md) is an optional collection of presets for Islandora, intended to provide a more user-friendly out-of-the-box experience and starting point for more specific customization. 
 
 Islandora Defaults provides:
 
-- a [Content Types](../user-documentation/content_types.md) "Repository Item" that uses the `field_member_of` field, so that users may add nodes of this type to a collection (or paged content, or compound resource),
-- logic (a [Context](../user-documentation/context.md)) such that if a resource is tagged as a "collection", a view of its members will show on the collection's page.
-
-
+- a [Content Type](../user-documentation/content_types.md) "Repository Item" that uses the `field_member_of` field, so that users may add nodes of this type to a collection (or paged content, or compound resource),
+- A [View](../tutorials/create_update_views.md) showing the members of the collection.
+- logic (a [Context](../user-documentation/context.md)) such that if a resource is tagged as a "Collection" (in the "Model" field, then a view of its members will show on the collection's page.
 
 For more details, see the tutorial on [How to create and add to a collection](../tutorials/how-to-create-collection.md)
 
 ## Bulk management of members of a collection
 
-Bulk management of items can be done using the Drupal contrib module [Views Bulk Edit](https://www.drupal.org/project/views_bulk_edit). In short, build a view using this module, and you will be able to perform Drupal Actions on sets of objects. Neither Islandora nor Islandora Defaults provide out-of-the-box management tools, but the sandbox provides some sample content and views that use Views Bulk Edit.
+Bulk management of items can be done from a Views Bulk Operations-compatible View, including the main Drupal Content View (at admin/content) and the Manage Members View (at node/[node]/members). This is possible due to the Drupal contrib modules [Views Bulk Operations (VBO)](https://www.drupal.org/project/views_bulk_operations) and [Views Bulk Edit](https://www.drupal.org/project/views_bulk_edit). VBO allows you to perform Drupal Actions on objects. These Actions include making Islandora Derivatives such as "Audio - Generate a Service File from an Original File" and also to perform Drupal core Actions such as publishing/unpublishing content. The Views Bulk Edit module extends VBO and allows you to edit field values (such as the Member Of field, which would change which collection a group of nodes are in). 
+
+For instance, if you want to move a number of nodes from one collection into another, then you can 
```

Comment:
The following list items don't show up as bullet points when I build the documentation page, I think they need to be indented in the source code